namelayer-spigot/src/main/java/vg/civcraft/mc/namelayer/permission/GroupPermission.java

```diff
@@ -65,33 +64,31 @@ public String listPermsforPlayerType(PlayerType type){
 	}
 	/**
 	 * Adds a PermissionType to a PlayerType.
-	 * @param pType- The PlayerType.
-	 * @param permType- The PermissionType.
+	 * @param pType The PlayerType.
+	 * @param permType The PermissionType.
 	 * @return Returns false if the PlayerType already has the permission.
 	 */
-	public boolean addPermission(PlayerType ptype, PermissionType permtype){
-		return addPermission(ptype,permtype,true);
+	public boolean addPermission(PlayerType pType, PermissionType permType){
+		return addPermission(pType, permType, true);
 	}
 	public boolean addPermission(PlayerType pType, PermissionType permType, boolean savetodb){
 		if (perms.get(pType).contains(permType))
 			return false;
 		List<PermissionType> types = perms.get(pType);
 		types.add(permType);
-		List <PermissionType> toAdd = new LinkedList<PermissionType>();
-		toAdd.add(permType);
```

Comment:
Wait what

Reply:
what's the problem

Reply:
I thought the list you inserted didnt have the permType anymore, but it does the syntax is just confusing. Nevermind